src/components/subscriptions/KeyMasker.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component, Fragment } from "react";
+import { WithNamespaces, withNamespaces } from "react-i18next";
+import MdVisibility from "react-icons/lib/md/visibility";
+import MdVisibilityOff from "react-icons/lib/md/visibility-off";
+
+type OwnProps = {
+  label: string;
+  value: string;
+  masked: boolean;
+  onMaskChange: (masked: boolean) => void;
+};
+type Props = WithNamespaces & OwnProps;
+
+class KeyMasker extends Component<Props> {
+  public render() {
+    const { t, masked, label, value } = this.props;
+
+    return (
+      <Fragment>
```

Comment:
Tip: `<Fragment>` can be just `<>`

Reply:
I know, but I prefer `<Fragment>` because:
- older versions of the JSX Babel plugin didn’t understand `<>`
- `<>` does not support `key`
- less readable and understandable